paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -1030,23 +1030,26 @@ PDNode *patterns::ElewiseAddActInplaceGrad::operator()(
 }
 
 PDNode *patterns::ConvBias::operator()(
-    paddle::framework::ir::PDNode *conv_input) {
+    paddle::framework::ir::PDNode *conv_input, bool is_conv3d) {
```

Comment:
```
std::string type = is_conv3d() ? "conv3d" : "conv2d";
```

Reply:
Done! This PR is not associated with [PR#14768](https://github.com/PaddlePaddle/Paddle/pull/14768).

Reply:
You can run "Analyzer_dam.compare_mkldnn" and check the below.
--- Running IR pass [**conv3d_bias_mkldnn_fuse_pass**]
---  **detect 2 subgraphs**
--- Running IR pass [conv_relu_mkldnn_fuse_pass]